src/network/network_connection.cpp

```diff
@@ -391,7 +391,7 @@ WriteState LibeventSocket::FlushWriteBuffer() {
   return WRITE_COMPLETE;
 }
 
-void LibeventSocket::PrintWriteBuffer() {
+void NetworkConnection::PrintWriteBuffer() {
```

Comment:
No "Print" methods. Rename it to `WriteBufferToString` and have it return `std::string`